File: Primer Trimestre/Examen Noviembre Prueba/Ejercicio2.java
/**
Haz un programa en Java y Python que calcule el elemento N 
(se pide al usuario) de la serie de Fibonacci.

@author: Rafael López Cruz
*/
import java.util.Scanner;
public class Ejercicio2 {
	public static void main(String[] args) {
		Scanner s = new Scanner(System.in);
		int n;
		
		//Pedimos numero de la serie de Fibonacci
		System.out.print("Introduce la serie de Fibonacci:");
		n = s.nextInt();
		
		//Declaro array
		int[] num = new int [n+1];
		num[0] = 0;
		num[1] = 1;
		
		//Ejecutamos bucle for que va a calcular el numero
		for (int i = 0; i <= n-2; i++) {
			num[i+2] = num[i]+num[i+1];
		}
		
		//Imprimimos
		System.out.println("El número de Fibonacci es "+ num[n-1]);
		}
}

File: Primer Trimestre/Examen Noviembre Prueba/Ejercicio2.py
"""
Haz un programa en Java y Python que calcule el elemento N 
(se pide al usuario) de la serie de Fibonacci.

__author__ = "Rafael López Cruz"
"""
import array
import numpy

#Pedimos numero de la serie de Fibonacci
n = (int(input("Introduce la serie de Fibonacci: ")))

#Declaro array
num = numpy.empty(n+1, dtype=object)
num[0] = 0
num[1] = 1

#Ejecutamos bucle for que va a calcular el numero
for x in range(0, n-2):
    num[x+2] = num[x]+num[x+1]

#Imprimimos
print("El numero de Fibonacci es",num[n-1])
    